crates/matrix-sdk/src/room/timeline/event_handler.rs

```diff
@@ -299,6 +308,18 @@ impl<'a> TimelineEventHandler<'a> {
         }
     }
 
+    fn handle_room_encrypted(&mut self, c: RoomEncryptedEventContent) {
+        match c.relates_to {
+            Some(encrypted::Relation::Replacement(_) | encrypted::Relation::Annotation(_)) => {
+                // Do nothing for these, as they would not produce a new
+                // timeline item when decrypted either
```

Comment:
Does this make sense?

Reply:
Could you expand on this? Is this about an encrypted event, that we can't decrypt, being a replacement that is an edit.

The assumption here is that edits only update timeline items but don't produce new ones, right? If that's so, then this sounds correct.

Reply:
Exactly. I was wondering whether we should show a UTD item for an edit that can't be decrypted (this branch also includes "annotations", i.e. reactions which are currently never encrypted so maybe that variant shouldn't exist at all 🤔).